### DISCUSSION TITLE
How to verify AnalyticsWithoutAdIdSupport is working correctly?

### DISCUSSION THREAD

**Discussion by canpoyrazoglu** [invertase/react-native-firebase#5617] [Q&A]
I want Firebase Analytics/Crashlytics but for legal reasons want to be 100% sure it's being linked with `AnalyticsWithoutAdIdSupport`. I've seen the PR https://github.com/invertase/react-native-firebase/pull/5224 and added the following to my podfile:

`$RNFirebaseAnalyticsWithoutAdIdSupport = true`

However when I `pod install` (or `pod update`) I see no change, and don't know how to verify if it worked or not. How can I verify if it's being to the correct library without the Ad ID support?

**Comment by mikehardy** [comment]
Inspection of Podfile.lock is the way to do it, you will see quite clearly, like this:

```
  - FirebaseAnalytics/WithoutAdIdSupport (8.4.0):
    - FirebaseCore (~> 8.0)
    - FirebaseInstallations (~> 8.0)
    - GoogleAppMeasurement/WithoutAdIdSupport (= 8.4.0)
    - GoogleUtilities/AppDelegateSwizzler (~> 7.4)
    - GoogleUtilities/MethodSwizzler (~> 7.4)
    - GoogleUtilities/Network (~> 7.4)
    - "GoogleUtilities/NSData+zlib (~> 7.4)"
    - nanopb (~> 2.30908.0)

```

**Reply by mikehardy** [reply]
(also, you may like all of the config power I just added for GDPR compliance - I'm not sure it was possible to 100% not touch remote services on startup / wait for user consent - prior to the most recent batch of changes #5600  / #5603 )
